bpfd-operator/README.md

```diff
@@ -1,80 +1,179 @@
 # bpfd-operator
-// TODO(user): Add simple overview of use/purpose
+
+The Bpfd-Operator repository exists in order to deploy and manage bpfd within a kubernetes cluster.
+
+![bpfd on K8s](./bpfd-on-k8s.jpg)
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
 
-## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+This repository houses two main processes, the `bpfd-agent` and the `bpfd-operator` along with CRD api definitions for `EbpfProgram` and `EbpfProgramConfig` Objects. In the below sections we'll dive a bit deeper into the API and functionality of both processes.
 
-### Running on the cluster
-1. Install Instances of Custom Resources:
+But first try it out!
 
-```sh
-kubectl apply -f config/samples/
+## Getting started
+
+This operator was build utilizing some great tooling provided by the [operator-sdk library](https://sdk.operatorframework.io/). A great first step in understanding some
+of the fucntionality can be to just run `make help`.
+
+After reviewing the possible targets it's quick and easy to get bpfd deployed locally on your system via a [KIND cluster](https://kind.sigs.k8s.io/). with:
+
+```bash
+make run-on-kind
 ```
 
-2. Build and push your image to the location specified by `IMG`:
-	
-```sh
-make docker-build docker-push IMG=<some-registry>/bpfd-operator:tag
+**NOTE**
```

Comment:
```suggestion
**NOTE**

```
Not sure if adding a new line does it, but **NOTE** is on the same line as the next sentence when the markdown is rendered. Otherwise, add a `:` to NOTE and leave it on the same line.

Reply:
Removed it :)